recipes/python.rb

```diff
@@ -0,0 +1,2 @@
+node.override['python']['pip_location'] = \
+"#{node['python']['prefix_dir']}/bin/pip2.7"
```

Comment:
Is Python 2.7 installed on the machine? From what you're saying, it sounds like it isn't but then I'm not sure you want to set pip to pip2.7.